stage/services/salesforce-api/salesforce-api_runner.js

```diff
@@ -1,3 +1,25 @@
+function hasBillingInfo(res = new Object({})) {
+  return (
+    res && res.attributes && (
+      res.attributes.billing_organization ||
+      res.attributes.billing_department ||
+      res.attributes.billing_street ||
+      res.attributes.billing_city ||
+      res.attributes.billing_state_code ||
+      res.attributes.billing_postal_code ||
+      res.attributes.billing_country_code
+    )
+  ) ? true : false;
+}
+
+function hasjoinedDate(res = new Object({})) {
+  return (
+    res && res.attributes && (
+      res.attributes.joined
+    )
+  ) ? true : false;
```

Comment:
I don't think this is necessary.

Reply:
Ye the extra function isn't needed here, you can just check res.attributes.joined directly.